libs/perun/components/src/lib/groups-list/groups-list.component.ts

```diff
@@ -129,6 +137,10 @@ export class GroupsListComponent {
     this.displayButtons = window.innerWidth > 800;
   }
 
+  ngOnInit(): void {
+    this.disabledRouting = this.disableRouting;
```

Comment:
I realized that this could also cause some problems. For example, in vo-groups there is routeAuth which is not executed until some async call is done. Now, the routing is disabled on this page because the property `[disableRouting]="!routeAuth"` is undefined.